dotcom-rendering/src/web/components/Card/components/TrailTextWrapper.tsx

```diff
@@ -28,7 +28,7 @@ export const TrailTextWrapper = ({
 				padding-right: 5px;
 				padding-bottom: 6px;
 
-				${until.tablet} {
+				${until.desktop} {
```

Comment:
I've added this here because it recreates the behaviour of this container on Frontend. However, it's the sort of change that will have implications for all other cards, too. If there are other requirements for other containers, we'll need to introduce different ways of setting this value for different containers. Any thoughts?

Reply:
I’m a little bit puzzled by this change. This has been introduced in 2019 in [PR #1019](https://github.com/guardian/dotcom-rendering/pull/1019/commits/d5566e2953ba4dbcee290b1b168a3178b84c054f) and I would like to push this to a separate PR if at all possible?

Reply:
+1 for a separate PR for this. This could have impacts in a wider area than just fronts.

Reply:
Thanks, I've taken it out of this branch now. Do you think it makes sense to add an issue treating  the current behaviour of the trail text as a bug because it diverges from Frontend fronts? Or should we treat this as the baseline now given that it seems to follow an established convention on DCR?